src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(numb) {
+      result += numb;
+    },
+
+    subtract(numb) {
+      result -= numb;
+    },
+
+    multiply(numb) {
+      result *= numb;
+    },
+
+    divide(numb) {
+      result /= numb;
```

Comment:
The `divide` method should handle division by zero to prevent `Infinity` or `NaN` results. Consider adding a check to ensure `numb` is not zero before performing the division.